data-explorer/follower.md

```diff
@@ -51,6 +54,12 @@ When attaching the database all tables, external tables and materialized views a
     tablesToInclude = []
     ```
 
+1. Include all functions. The '*' is needed, since none followers are followed by default:
+
+    ```kusto
+    functionsToInclude = ["*"]
+    ```
+
```

Comment:
```suggestion
1. Include all functions. No '*' is needed, since all tables are followed by default:

    ```kusto
    functionsToInclude = [ ]
    ```

```

Reply:
Just fixed, thanks!